### ISSUE TITLE
[KED-1642] `index` option in `pandas.ParquetDataSet`

### ISSUE THREAD

**Issue by juan-carlos-calvo** [quantumblacklabs/kedro#352]
## Description
`ParquetLocalDataSet` accepts the `index` save option which specifies whether to save the index in parquet or not. `pandas.ParquetDataSet` doesn't have an equivalent option at the moment.

## Context
As the datasets in `kedro.io` are going to be deprecated and moved to `kedro.extras.datasets` kedro should have backward compatibility with the `load_args` `save_args` options. Also, index is a useful option to have.

## Possible Implementation
at the `_save` method of `pandas.ParquetDataSet` substitute:
```
table = pa.Table.from_pandas(data)
```
with
```
preserve_index = self._save_args.pop('index', False)
table = pa.Table.from_pandas(data, preserve_index=preserve_index)
```

**Comment by 921kiyo** [comment]
@juan-carlos-calvo Thank you for opening the issue! This makes sense to me. I've logged this in our backlog and will fix it :)

**Comment by andrii-ivaniuk** [comment]
Thanks @juan-carlos-calvo for reporting this.
It was fixed in 5330b45ce02b61281a27698a7fb67a2e09decaef commit.
